out

```diff
@@ -0,0 +1,75 @@
+diff --git a/auto_merge_201711171113:lib/CodeGen/CodeGenFunction.h b/clang/amd-common:lib/CodeGen/CodeGenFunction.h
```

Comment:
This file shouldn't be merged into hcc_clang_upgrade's clang_tot_upgrade?

Reply:
it should not.  It is a left over artifact from investigating the merge conflict, that I forgot to remove.